doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -143,7 +143,13 @@ DECT NR+
 Enhanced ShockBurst (ESB)
 -------------------------
 
-|no_changes_yet_note|
+* Added:
+
+  * :kconfig:option:`CONFIG_ESB_NEVER_DISABLE_TX` Kconfig option support for packets with ACK.
+
+* Updated:
+
+  * Renamed :kconfig:option:`CONFIG_ESB_NEVER_DISABLE_TX` to :kconfig:option:`CONFIG_ESB_KEEP_TX_IDLE`.
```

Comment:
This does not make much sense. You first add CONFIG_ESB_NEVER_DISABLE_TX and then rename it.

Reply:
I updated the changelog.